sycl/include/CL/sycl/xilinx/fpga/opt_decorate_func.hpp

```diff
@@ -59,7 +53,7 @@ void dataflow(T functor) {
     pipeline way.
 */
 template <typename T>
-SYCL_DEVICE_ANNOTATE("xilinx_pipeline") __attribute__((always_inline))
+__SYCL_DEVICE_ANNOTATE("xilinx_pipeline") __attribute__((always_inline))
```

Comment:
Could even be `__TRISYCL` prefix to reduce conflicts with Intel or other SYCL implementations